readme.md

```diff
@@ -281,13 +281,13 @@ This small tool lets you start TensorBoard without having to type in all the log
 
 To get a list of all available experiments
 ```bash
-LAB/tb.py -l
+tensorboard.py -l
```

Comment:
should it be `[LAB]/tensorboard.py`

Reply:
nope, running it in the lab's root directory worked for me. But it could be due to me running this on the samples which were in the default location.

Reply:
no you need to run it from your project (which is lab if you are trying samples)